ranger-core/src/main/java/io/appform/ranger/core/model/ServiceNode.java

```diff
@@ -30,8 +30,13 @@
     private String host;
     private int port;
     private T nodeData;
+    @Builder.Default
     private HealthcheckStatus healthcheckStatus = HealthcheckStatus.healthy;
+    @Builder.Default
     private long lastUpdatedTimeStamp = Long.MIN_VALUE;
+    //Can be any scheme that you intend to init.
+    @Builder.Default
+    private String scheme = "http";
```

Comment:
* rename this to `portScheme`.
* create a class called `PortSchemes` that has string constants like HTTP, HTTPS, TCP etc with values "http", "https", "tcp" respectively
* set portScheme deffault to portScheme = PortSchemes.HTTP

Reply:
Booted with the same constants. Resolving this!